source/fuzz/transformation_replace_branch_from_dead_block_with_exit.h

```diff
@@ -48,6 +48,10 @@ class TransformationReplaceBranchFromDeadBlockWithExit : public Transformation {
   // - If |message_.opcode()| is OpReturnValue then |message_.return_value_id|
   //   must be an id that is available at the block terminator and that matches
   //   the return type of the enclosing function
+  // - Domination rules should be preserved when we apply this transformation.
```

Comment:
Can you add "if" in "can only be OpKill [if] the module's entry points" above (as part of this PR)